config/components/config/holiday.py

```diff
@@ -0,0 +1,12 @@
+from django_components import Component, register
+
+
+@register("config.holiday_table")
+class HolidayTable(Component):
+
+    # This component takes one parameter, a date string to show in the template
```

Comment:
Haha, this comment is still getting copy-pasted :)

Reply:
right well got rid of it ;)